aws/resource_aws_volume_attachment_test.go

```diff
@@ -329,10 +329,40 @@ data "aws_availability_zones" "available" {
   }
 }
 
+data "aws_ami" "amzn-ami-minimal-hvm-ebs" {
+  most_recent = true
+  owners      = ["amazon"]
+
+  filter {
+    name   = "name"
+    values = ["amzn-ami-minimal-hvm-*"]
+  }
+
+  filter {
+    name   = "root-device-type"
+    values = ["ebs"]
+  }
+}
+
+data "aws_ec2_instance_type_offering" "available" {
```

Comment:
Maybe split this into a separate config generating function that takes a list of potential instance types?
This will be useful in many places.